torch/autograd/gradcheck.py

```diff
@@ -61,6 +61,24 @@ def get_numerical_jacobian(fn, input, target=None, eps=1e-3):
     x_tensors = iter_tensors(target, True)
     j_tensors = iter_tensors(jacobian)
 
+    def compute_gradient(x, idx, is_mkldnn=False):
+
+        def fn_out(is_mkldnn=False):
```

Comment:
Why do you need to pass this as an argument? You can just capture it from above?

Reply:
agreed will remove it